spec/fixtures/partial-use-network.html

```diff
@@ -0,0 +1,18 @@
+<div>
```

Comment:
Can we rename this file to `partial-allow-from-url.html` like the name of the plugin option?